lib/screens/admin_order_detail_page.dart

```diff
@@ -0,0 +1,585 @@
+          _buildShippingMethodInfoBox(),
+        ],
+      ),
+    );
+  }
+
+  Widget _buildGreyBar() {
+    return Container(
+      padding: EdgeInsets.only( top: 24.0 ),
+        child: Container(
+        decoration: BoxDecoration(
+          color: const Color(0xFFEFEFEF),
+          borderRadius: BorderRadius.all( Radius.circular(40) ),
+        ),
+        height: 8,
+        width: 115,
+      ),
+    );
+
+  }
+
+  Widget _buildBuyerContactInfoBox() {
+    return Container(
+      padding: EdgeInsets.only(
+        top: 12.0, left: 24.0, right: 24.0, bottom: 36.0
+      ),
+      child: StreamBuilder<DocumentSnapshot>(
+        stream: Firestore.instance
+            .collection(constants.DBCollections.users)
+            .document(widget.order.customerId)
+            .snapshots(),
+        builder: (context, snapshot) {
+          if (!snapshot.hasData) {
+            return LinearProgressIndicator();
+          }
+          Customer buyer = Customer.fromSnapshot(snapshot.data);
+          return Column(
+            children: <Widget>[
+              Container(
+                alignment: Alignment.center,
+                child: Text(
+                  "注文者情報",
+                  style: TextStyle(
+                    fontSize: 18, color: paletteBlackColor, fontWeight: FontWeight.bold,
+                  )
+                ),
+              ),
+              SizedBox(height: 24.0),
+              _buildBuyerName(buyer),
+              SizedBox(height: 12.0),
+              _buildBuyerAddress(buyer),
+              SizedBox(height: 6.0),
+              _buildBuyerPhone(buyer),
+              SizedBox(height: 6.0),
+              _buildBuyerEmail(buyer),
+            ],
+          );
+        }
+      )
+    );
+  }
+
+  Widget _buildBuyerName(Customer buyer) {
+    return Container(
+      alignment: Alignment.centerLeft,
+      child: Row(
+        children: <Widget>[
+          Text(
+            buyer.name,
+            style: TextStyle(
+              fontSize: 16,
+              color: paletteBlackColor,
+              fontWeight: FontWeight.w600,
+            )
+          ),
+          Text(
+            " 様",
+            style: TextStyle(
+              fontSize: 16,
+              color: paletteBlackColor,
+              fontWeight: FontWeight.w300,
+            )
+          ),
+        ],
+      ),
+    );
+  }
+
+  Widget _buildBuyerAddress(Customer buyer) {
+    return Container(
+      child: Row(
+        crossAxisAlignment: CrossAxisAlignment.start,
+        children: <Widget>[
+          Text(
+            "住所",
+            style: TextStyle(
+              fontSize: 16, color: paletteBlackColor, fontWeight: FontWeight.w300,
+            ),
+          ),
+          SizedBox(width: 30.0),
+          Expanded(
+            child: Column(
+              crossAxisAlignment: CrossAxisAlignment.start,
+              children: <Widget>[
+                Text(
+                  // buyer.shippingAddresses[0].postCode
```

Comment:
Can you refer to the actual `buyer.shippingAddresses[0].postCode` instead of the hard coded text? You can add this dummy data to the customer model in the database from the Firebase console.